tools/depends/Makefile.include.in

```diff
@@ -44,10 +44,12 @@ CONFIG_GUESS=@prefix@/@tool_dir@/share/automake-1.15/config.guess
 
 RPL=@prefix@/@tool_dir@/bin/python @prefix@/@tool_dir@/bin/rpl
 
+USE_CCACHE=@use_ccache@
+
 LD=@LD@
-CC=@CC@
-CXX=@CXX@
-CPP=@CPP@
+CC=@CCACHE@ @CC@
+CXX=@CCACHE@ @CXX@
+CPP=@CCACHE@ @CPP@
```

Comment:
@wsnipex does this also apply to `native/config.site.native.in` and `target/config.site.in`?